events/index.html

```diff
@@ -41,39 +41,55 @@
 
 {% block content_main %}
 
-    {# TEMP The mocked event content is in the following paragraph #}
-    <p>
-        Events content goes here: {{vars.mock_post.content}}
-    </p>
+    <section class="media block block__flush-top block__sub">
+        <div class="media_body">
+            <h1>Upcoming events</h1>
```

Comment:
Not to hold each of these up b/c of semantics, but you might consider moving heading content outside of a `<div>` with a name of `media_body`, `<h1>Upcoming events</h1>` will become the title for  `<section class="media block block__flush-top block__sub">` in the document outline.